README.md

```diff
@@ -2,15 +2,21 @@
 
 Simple client-side library exported as a JavaScript package, for working with ontologies.
 
+This is a monorepo, which contains sub-packages:
+* [packages/OntologyService/README.md](packages/OntologyService/README.md)
+* [packages/RdfService/README.md](packages/RdfService/README.md)
+
 ## Install
 
+This monorepo's sub-packages are published to https://www.npmjs.com/org/telicent-oss:
+
 ```sh
-cd <yourProject>
 yarn install @telicent-oss/ontologyservice
 ```
 
 ## Usage
 
+A simple example:
```

Comment:
That does what?  I'm guessing from the code fragment that it gets all diagrams from the ontology service.  If I'm correct then this line should read:
"A simple example to get all diagrams from the ontology service:"